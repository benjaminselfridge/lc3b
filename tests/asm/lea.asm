        0x4000

START:  LEA R0 DATA
        ADD R0 R0 R0
        TRAP 0x0

DATA:   0x1234                  ;0x4006
