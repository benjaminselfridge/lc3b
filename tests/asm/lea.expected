[ RegContains 0 0x800c ]